openmetadata-ui/src/main/resources/ui/src/styles/app.less

```diff
@@ -429,8 +429,6 @@ a[href].link-text-grey,
   }
 }
 .entity-tag-right-panel-container {
-  height: calc(100vh - 236px);
-  overflow-y: scroll;
```

Comment:
Removing this as added by this PR https://github.com/open-metadata/OpenMetadata/pull/12163 and we don't need it.